plugin_interpreter/src/controller_plugin.py

```diff
@@ -224,7 +231,7 @@ def _advertise_functionality(self):
         """
         self.DBI.create_plugin_table((self.name, self.functionality))
 
-    def _request_job(self):
```

Comment:
API changes usually deprecate over at least 1 sprint.

Suggest leaving _request_job with a stderr.write that it's now deprecated

Add a task to remove in a future sprint.